src/components/MenuContent/MenuContent.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ComponentType } from 'react';
+import { menuContent } from 'cms-content/footer';
+import FeaturedSection from './FeaturedSection';
+import LearnSection from './LearnSection';
+import AboutUsSection from './AboutUsSection';
+import { ContentWrapper } from './Menu.style';
+import { useBreakpoint } from 'common/hooks';
+import {
+  Experiment,
+  ExperimentID,
+  Variant,
+  VariantID,
+} from 'components/Experiment';
+import { FeaturedItem, SectionId } from 'cms-content/footer';
+
+const FeaturedSectionVariant: React.FC<{
+  featuredSections: FeaturedItem[];
+  onClick: (label: string) => void;
+}> = ({ featuredSections, onClick }) => {
+  const apiLink = featuredSections.filter(
+    section => section.iconId === SectionId.API,
+  );
+  const dailyDownloadLink = featuredSections.filter(
+    section => section.iconId === SectionId.DAILY_DOWNLOAD,
+  );
+  const alertsLink = featuredSections.filter(
+    section => section.iconId === SectionId.ALERTS,
+  );
+
+  const orderedSectionsA = [...apiLink, ...alertsLink, ...dailyDownloadLink];
+  const orderedSectionsB = [...alertsLink, ...dailyDownloadLink, ...apiLink];
+
+  return (
```

Comment:
These 2 arrays (and the code above) can be moved outside the render method, since they don't need to recalculated each time the component renders

Reply:
as in into the cms index? they use the `featuredSections` prop

Reply:
its a pretty small calculation- im tempted to just keep as is and 🚢